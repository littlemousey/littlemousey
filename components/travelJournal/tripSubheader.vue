<template>
  <v-layout row wrap>
    <v-flex xs12>
      <div class="display-1">
        {{ title }}
      </div>
    </v-flex>
  </v-layout>
</template>

<script>
export default {
  name: 'TripSubheader',
  props: { title: { type: String, default: '' } }
}
</script>
