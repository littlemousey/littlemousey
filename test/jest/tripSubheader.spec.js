import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import tripSubheader from '@/components/travelJournal/tripSubheader'

Vue.use(Vuetify)

let wrapper
beforeEach(() => {
  wrapper = shallowMount(tripSubheader)
})

afterEach(() => {
  wrapper.destroy()
})

describe('Trip subheader', () => {
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
