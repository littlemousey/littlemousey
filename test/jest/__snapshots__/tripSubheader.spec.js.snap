// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trip subheader renders correctly 1`] = `
<v-row-stub
  tag="div"
>
  <v-col-stub
    cols="12"
    tag="div"
  >
    <h3
      class="display-1 my-4 text-underline"
    >
      
      
    
    </h3>
  </v-col-stub>
</v-row-stub>
`;
